skin/src/main/java/org/researchstack/skin/DataProvider.java

```diff
@@ -105,7 +106,7 @@ public static void init(DataProvider instance)
      * @return Observable of the result of the method, with {@link DataResponse#isSuccess()}
      * returning true if signIn was successful
      */
-    public abstract Observable<DataResponse> resendEmailVerification(Context context, String email);
```

Comment:
If you keep the method with the password parameter, please explain the situation thoroughly in the method header and designate the password parameter as @Nullable